Marlin/language_en.h

```diff
@@ -261,6 +261,24 @@
 #ifndef MSG_ESTEPS
 #define MSG_ESTEPS                          "Esteps/mm"
 #endif
+#ifndef MSG_X1OFFSET
+#define MSG_X1OFFSET                         "T1X Offset"
```

Comment:
Interesting direction this discussion went. :-)

I wanted to talk about text read by humans in on the display, what has not necessary to be similar to a related g-code. And i wanted to talk about the efficiency of memory usage.

We already are in a sub-menu called "Extruder offset:". Should we now change to T.. for the menu-items? Maybe "T" is simply to short to be expressive. How about "Tool"?
If using "T" or "Tool" the sub-menu's title should be "Tool offset".

If it makes sense to split `"T1X Offset"` to `"T1X" " Offset"` or  `"T1X" MSG_OFFSET` entirely depends on the question when(where) the concatenation is done. I suppose all versions will end as a PSTR("T1X Offset") because the concatenation is done by the preprocessor and no memory is saved. But here is not the right place to discuss this.

For short. I suggest to ether rename the the menu-title or the menu-items.

The 'T' as a parameter-name for the g-code is ok for me, since the hosts/slices are used to this usage now. (If a eventually coming "standard" will forbid this usage for 'T' we can change this - but i don't believe on that.)